test/cases/api.sh

```diff
@@ -438,6 +439,39 @@ EOF
 )
 export FIREWALL_CUSTOMIZATION_BLOCK
 
+if [ "$TEST_MODULE_HOTFIXES" = "1" ]; then
+  if [ "$ARCH" = "x86_64" ]; then
+    NGINX_REPO_URL="https://rpmrepo.osbuild.org/v2/mirror/public/el8/el8-x86_64-nginx-20231207"
+  else
+    NGINX_REPO_URL="https://rpmrepo.osbuild.org/v2/mirror/public/el8/el8-aarch64-nginx-20231207"
+  fi
+  EXTRA_PAYLOAD_REPOS_BLOCK=$(cat <<EOF
+,
+      {
+        "baseurl": "$NGINX_REPO_URL",
+        "check_gpg": false,
+        "check_repo_gpg": false,
+        "rhsm": false,
+        "module_hotfixes": true
+      }
+EOF
+)
+
+  EXTRA_PACKAGES_BLOCK=$(cat <<EOF
+,
+      "nginx",
+      "nginx-module-njs"
```

Comment:
I realized that it would be good to extend https://github.com/osbuild/osbuild-composer/blob/9b53d3e6a66ae8f2dd9845a160475a20f1617161/test/cases/api/common/common.sh#L18-L78 to check for these packages on the booted image, in case `[ "$TEST_MODULE_HOTFIXES" = "1" ]`

A follow-up is OK.

Reply:
given I'm not sure how to do this, I'd rather work on it separatelly, so we can work on image builder integration in parallel

Reply:
It would be very similar to the code on line 23...

Reply:
gotcha, it will take few trials tho, so I'll open a follow-up for this: https://github.com/osbuild/osbuild-composer/pull/3863